frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -532,6 +534,32 @@ const plugin: Plugin<ConsumedExtensions> = [
         ).default,
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: ['/samples/all-namespaces', '/samples/ns/:ns'],
+      loader: async () =>
+        (
+          await import(
+            './components/import/SamplesCatalog' /* webpackChunkName: "dev-console-samples-catalog" */
+          )
+        ).default,
+    },
+  },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: ['/samples/ns/:ns/is/:is/isNs/:isNs'],
```

Comment:
Why the intermediate segments `is` and `isNs`? Seems unnecessary as you're not going to clash with any other route.

Reply:
Do you mean I should have the route like `/samples/:ns/:is/:isNs`?